src/libmr_commands.c

```diff
@@ -14,7 +14,11 @@
 static inline bool check_and_reply_on_error(ExecutionCtx *eCtx, RedisModuleCtx *rctx) {
     size_t len = MR_ExecutionCtxGetErrorsLen(eCtx);
     if (unlikely(len > 0)) {
+        RedisModule_ReplyWithArray(rctx, len + 1);
```

Comment:
Also lets wait for @danni-m with this change because you said he was against multiple errors.

Reply:
@MeirShpilraien Can we merge it just to check what is the problem on the CI and later revert it?
Also we need to tag 1.6.9 and it's a blocker, @gkorland can it wait for Monday?